src/CreditCardForm.js

```diff
@@ -78,6 +76,23 @@ export default class CreditCardForm extends Component {
 
     /** Optional API Options **/
     apiOptions: PropTypes.object,
+
+    /** a string that indicated the destination of the operation */
+    model: PropTypes.string,
+
+    /** a token that grants permission to interact with the RevOps API */
+    accessToken: PropTypes.string,
+
+    children: PropTypes.element,
+
+    /** model for of a revops instrument */
+    instrument: PropTypes.object,
+
+    /** getToken (accountId) => { access_token } callback function that is called before every call requiring authorization */
```

Comment:
Suggestion for writing this in a more terse way: 

`callback function that is called before every call requiring authorization` ->
`callback to request authorization token before every API request`

Reply:
Maybe add: "Why do I need to set this?"